CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # Changelog
 
+## streamlink 2.2.0 (2021-06-18)
+
+Release highlights:
+
+- Changed: default config file path on macOS and Windows ([#3766](https://github.com/streamlink/streamlink/pull/3766))  
+  - macOS: `${HOME}/Library/Application Support/streamlink/config`
+  - Windows: `%APPDATA%\streamlink\config`
+- Changed: default custom plugin directory path on macOS and Linux/BSD ([#3766](https://github.com/streamlink/streamlink/pull/3766))
+  - macOS: `${HOME}/Library/Application Support/streamlink/plugins`
+  - Linux/BSD: `${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins`
+- Deprecated: old config file paths and custom plugin directory paths ([#3784](https://github.com/streamlink/streamlink/pull/3784))  
```

Comment:
@bastimeyer I think the changelog should be more clear that we are in the process of deprecating the old paths and they will be deprecated on the next major release. This makes it seem like we're dropping support for them today.

Reply:
https://github.com/streamlink/streamlink/compare/fd00fc42b7b26f01f4176821f577d84661e5e65e..dac69bbb70e5285a7a3183c25fd395292d53b051

Reply:
If you want, we can explicitly list the deprecated paths, but as said, I'm not sure if it's too much. In the current version, it doesn't mention `~/.streamlinkrc` for example.

Reply:
What do you think about this? Is it too much?

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 7c7ef2ae..fc8b7a8f 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,16 @@ Release highlights:
   - macOS: `${HOME}/Library/Application Support/streamlink/plugins`
   - Linux/BSD: `${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins`
 - Deprecated: old config file paths and old custom plugin directory paths ([#3784](https://github.com/streamlink/streamlink/pull/3784))  
+  - Windows:
+    - `%APPDATA%\streamlink\streamlinkrc`
+  - macOS:
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/config`
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/plugins`
+    - `${HOME}/.streamlinkrc`
+  - Linux/BSD:
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/plugins`
+    - `${HOME}/.streamlinkrc`
+
   Support for these old paths will be dropped in the future.  
   See the [CLI documentation](https://streamlink.github.io/cli.html) for all the details regarding these changes.
 - Implemented: `--logfile` CLI argument ([#3753](https://github.com/streamlink/streamlink/pull/3753))
```

Reply:
__